pkg/script/kubeadm-join.go

```diff
@@ -0,0 +1,25 @@
+package script
+
+import "bytes"
+
+const kubeadmJoinTmpl string = `#!/bin/sh
```

Comment:
I'm a little bit confused. Why you put one script there, in the code, while the another are still in separate files? Can't we template scripts which are in separate files?

Reply:
@nhlfr Right. Scripts should be basically generated by `pkg/script` to be written to `/var/lib/kube-spawn/scripts`, and scripts under `./scripts` of the source tree, such as bootstrap, init or join, should be removed. They are just leftovers. I'll clean them up.